cmd/helios-cli/main.go

```diff
@@ -12,7 +12,6 @@
```

Comment:
Thanks for adding "version" to usage. Please mirror these flags in README so users discover them (`version`, `--version`, `-v`).